docs/content/patterns/helpful-form-content/index.mdx

```diff
@@ -0,0 +1,193 @@
+
+Present requirements as a scannable list of specific items. Use plain language to ensure a good understanding of what they will need.
+
+## Hint text
+
+Use hint text for the smallest piece of information that will help users to understand each form input or control and complete forms. This could be to mark a field as optional or to let them know the required format for a field.
+
+The hint prop provides help that’s relevant to the majority of users, like how their information will be used, or where to find it.
+
+Don’t use long paragraphs or lists in hint text. Screen readers announce the entire text when users interact with the form element. This could frustrate users if the text is long.
+
+Don’t include links within hint text. While screen readers will read out the link text when describing the field, they will not tell users that the text is a link.
+
+```jsx live
+<TextInput
+	label="Phone number"
+	hint="Any Australian mobile or landline. For example, 0444 111 222 or 02 9988 7766"
+/>
+```
+
+## Details
+
+The Details component reveals more information when opened. Use the Details component for information that is useful for some users, but not needed by everyone to complete a form. Do not use the Details component if all users need the information to complete a form.
+
+Details should only be used for one piece of content. We recommend keeping this to a paragraph or short list. If more than one piece of content is required, use a Drawer.
+
+If the guidance pertains to a list, provide the list, then link to more detailed information on all items.
+
+We recommend the use of one link to give a clear path to the right information for the user’s context. Links to content pages should be opened in the same tab. If a user is in the middle of a process, such as filling in a form, opening links into the same tab can cause a user to lose their place. In this case it’s preferable to open a link into a drawer or new tab. Clearly indicate this with an icon: `ExternalLinkIcon` for new tabs and `MoveLeftIcon` for drawers. Provide appropriate announcements for screen readers.
+
+```jsx live
+() => {
+	const [isDrawerOpen, openDrawer, closeDrawer] = useTernaryState(false);
+	return (
+		<>
+			<Details iconBefore label="Why should I invite a new staff member?">
+				<Flex flexDirection="column" gap={1}>
+					<Text as="p">
+						Inviting individuals to the service can help you streamline your
+						operations. You can then assign them a role, so they can access
+						necessary information and tasks.
+					</Text>
+					<Text as="p">
+						Learn more about{' '}
+						<Button
+							iconAfter={MoveLeftIcon}
+							onClick={openDrawer}
+							size="sm"
+							variant="text"
+						>
+							inviting a staff member
+						</Button>
+					</Text>
+				</Flex>
+			</Details>
+			<Drawer
```

Comment:
But yeah, we can't really have a working example like this because that's not how you'd use it… 🤔